dags.yaml

```diff
@@ -1542,3 +1542,17 @@ bqetl_acoustic_suppression_list:
     retry_delay: 30m
   tags:
     - impact/tier_3
+
+bqetl_braze:
+  schedule_interval: 0 9 * * *
+  description: |
+    ETL for Braze workflows.
+  default_args:
+    owner: cbeck@mozilla.com
+    email:
+      - cbeck@mozilla.com
+    start_date: '2024-04-20'
```

Comment:
don't we want to start running those models now?

Reply:
especially since I learned last week that we can't even backfill for dates before the start-date

Reply:
Yes ✅